### PR TITLE
Removed network calls from surrenderChance

### PR DESCRIPTION
**Pull request by Landric**
Addresses #88, should also increase performance and decrease network overhead, for minimal change in functionality

### CODE REVIEW

File: CrowsZA/functions/fn_surrenderChance.sqf [hunk 1]
```diff
@@ -77,18 +80,17 @@ _trigger attachTo [_unit];
...
-			remoteExecutedOwner
-		];
-	}] remoteExecCall [""call"", _x];
-
-	if((missionNamespace getVariable (""crowsza_surrender_chance_"" + str (_unit call BIS_fnc_netId)))) exitWith {true};
-} forEach thisList;
-(missionNamespace getVariable (""crowsza_surrender_chance_"" + str (_unit call BIS_fnc_netId)))
+_weaponsOnTarget = {
+   if(!isPlayer _x || {weaponLowered _x}) exitWith { 0 };
+   	_player = _x;
```
Comment (Crowdedlight) [inline CrowsZA/functions/fn_surrenderChance.sqf:85]: Seems to have one indent too much?

**Comment by Crowdedlight** [comment]
Looks good to me. Lets get it tested on dedicated to ensure there is no MP trickery and then get it merged.

**Comment by Crowdedlight** [comment]
@Landric Let me know if you wanted to add the extra bits on it as we talked about today, or its fine to merge now? 
Otherwise let me know when its ready to merge :)

**Comment by Landric** [comment]
Fine to merge now - I'll add the other bits as a separate update later down the line!

**Pull request merged**